src/app/frontend/common/components/zerostate/zerostate.html

```diff
@@ -17,9 +17,15 @@
 <kd-content-card>
   <kd-content>
     <div class="kd-zerostate-message" layout-padding>
-      <div class="kd-zerostate-title">{{::$ctrl.i18n.MSG_ZERO_STATE_TITLE}}</div>
+      <div class="kd-zerostate-title">[[There is nothing to display here|Title text which
+        appears on zero state view.]]</div>
       <div class="kd-zerostate-text">
-        <span ng-bind-html="::$ctrl.i18n.MSG_ZERO_STATE_TEXT"></span>
+        <span>
+          [[You can <a href="{{::$ctrl.getStateHref()}}">deploy a containerized app</a>,
+          select other namespace or
+          <a href="http://kubernetes.io/docs/user-guide/ui/" target="_blank">take the Dashboard Tour
+          <i class="material-icons kd-zerostate-icon">open_in_new</i></a>
+          to learn more| Description for a zero state page.]]</span>
```

Comment:
The variables feel very natural now! (to reiterate my excitement from before :D) That's probably the best part for me personaly, great stuff.